javatests/arcs/core/data/expression/ExpressionTest.kt

```diff
@@ -20,8 +20,21 @@ import org.junit.runners.JUnit4
 /** Tests for [Expression]. */
 @RunWith(JUnit4::class)
 class ExpressionTest {
-    fun <T> evalBool(expression: Expression<T>) = evalExpression<T, Boolean>(expression)
-    fun <T> evalNum(expression: Expression<T>) = evalExpression<T, Number>(expression)
+    fun <T> evalBool(expression: Expression<T>) = evalExpression<T, Boolean>(
+        expression,
+        currentScope
+    )
+
+    fun <T> evalNum(expression: Expression<T>) = evalExpression<T, Number>(
+        expression = expression, currentScope = currentScope
+    )
+
+    val currentScope = CurrentScope<Any>(
```

Comment:
maybe it could be:
```
    val currentScope = scopeOf(
        "blah" to 10,
        "baz" to scopeOf("x" to 24)
    )
```

Reply:
Nice idea, i'll save that for a followup, since we'll need to flesh out Builders.kt for full declarative AST nodes